src/main/java/Command.java

```diff
@@ -0,0 +1,25 @@
+import java.io.IOException;
+
+/**
+ * Represents a command to be executed.
+ */
+public abstract class Command {
+    protected String str;
+    protected boolean isExit = false;
+
+    /**
+     * Creates a command.
```

Comment:
Is it better if an empty line is placed separate the description and the param explanation?